server/lib/routes/get-config.js

```diff
@@ -48,7 +52,14 @@ module.exports = function () {
       language: req.lang,
       metricsSampleRate: metricsSampleRate,
       profileUrl: profileServerUrl
-    });
+    };
+
+    // only set the Sentry sample rate if API Key and endpoints are present
+    if (sentryAPIKey && sentryEndpoint) {
```

Comment:
What do you think about setting `sentrySampleRate` as long as the `sentrySampleRate` config option is available, and we can drop messages on the ground on the backend if they are not set. At a minimum, it will give devs the ability to see JS errors in their server console w/o actually calling Sentry.